plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsExt.kt

```diff
@@ -0,0 +1,102 @@
+@file:JvmName("CameraAnimationsUtils")
+package com.mapbox.maps.plugin.animation
+
+import com.mapbox.maps.CameraOptions
+import com.mapbox.maps.ScreenCoordinate
+import com.mapbox.maps.plugin.delegates.MapPluginExtensionsDelegate
+import com.mapbox.maps.plugin.delegates.MapPluginProviderDelegate
+
+/**
+ * Extension val for MapView to get the Camera Animations plugin instance.
+ */
+val MapPluginProviderDelegate.camera: CameraAnimationsPlugin
+  @JvmName("getCameraAnimationsPlugin")
```

Comment:
why so long naming?
perhaps just `camera()`?

Reply:
I'm calling it as you named it :) I'm fine with `getCamera()`